parm/config/gfs/config.stage_ic

```diff
@@ -26,6 +26,12 @@ case "${CASE}" in
     export CPL_OCNIC=HR1_refactored
     export CPL_WAVIC=HR1_refactored
     ;;
+  "C1152")
+    export CPL_ATMIC=HR2_refactored
```

Comment:
This will need to be something else. We've created new directories that reorganize the prototype directories in a cleaner way that also supports ensembles.
```suggestion
    export CPL_ATMIC=HR2_C1152_refactored
```